README.md

```diff
@@ -1 +1,106 @@
+
+Below this, you can select the Gen1 pokemon, each of them displaying their name as well as a small image next to it.
+
+![screenshot](./src/assets/images/screenshots/pokemon-lists.png)
+
+Once you click back on the highlighted badge the filter will clear and the full list of pokemon will show again.
+
+Then you click on any of the pokemon listed, you will be taken to the information page for that pokemon, where you can see a bigger image of it and the information about its types, weight, and height.
+
+![screenshot](./src/assets/images/screenshots/pokemonlistdetails.png)
+
+By clicking the home button on the home page you will go to the list that you had before, either if it was the full list or a filtered one.
+
+## Setup Locally
+
+To run the application locally you will need to follow several steps to make it work.
+
+First, you have to check if you have **nodejs** installed on your machine. For this type the following command in your terminal:
+
+> node -v
+
+If **command not found** shows as a response, then go to [nodejs.org](https://nodejs.org/en/) and follow the instructions to install it. After completing the installation the same command should show you a version number.
+
+Then clone the [repository](https://github.com/Kumilachew-g/react-capstone.git) into your local drive and open a terminal in the root folder of the project and install the necessary packages to make it work by running this command:
+
+> npm install
+
+This will get the project ready to run. After this run the following command to start the development server on **localhost:3000/** and wait for the browser to open:
+
+> npm start
+
+If the browser does not open or you close it by accident, but the server is still running you can type **localhost:3000/** in your address bar to get into the application again.
+
+## Tests
+
+This project has a suite of tests built-in.
+
+To run the tests clone the [repository](https://github.com/Kumilachew-g/react-capstone.git) into your local drive, if you haven't done so, and open a terminal in the root folder of the project and install the necessary packages to make it work by running this command:
+
+> npm install
+
+Then you can just run the following command and see the results
+
+> npm test
+
+## Author
+
+👤 **Kumilachew Getie**
+
+- GitHub: [@githubhandle](https://github.com/Kumilachew-g/)
+- Twitter: [@twitterhandle](https://twitter.com/Getie_Haddis)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/kumilachew-getie/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Kumilachew-g/react-capstone/issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse
+- [PokeAPI](https://pokeapi.co/docs/v2)
+- [The-Pokemon-Company](https://www.pokemon.com/us/)
+- [Netlify](https://www.netlify.com/)
+- [Dave Ceddia](https://www.youtube.com/channel/UCPrkHuq0i1WflifRXUpvNmg)
```

Comment:
as a requirement for this project, your README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 
👍🏾 👍🏾